resources/views/content/scripts.twig

```diff
@@ -1,18 +1,20 @@
 <script type="text/javascript">
-    document.addEventListener('flAutocompleteReady', function (e) {
-        var originalSelectItem = e.detail.flAutocomplete.selectItem;
-        e.detail.flAutocomplete.selectItem = function (event, ui) {
-            originalSelectItem.call(e.detail.flAutocomplete, event, ui);
+    window.flSearchSubmitEvent = document.createEvent('Event');
+    window.flSearchSubmitEvent.initEvent('fl-search-submit', true, true);
```

Comment:
Use up-to-date method for events since `initEvent` is deprecated

Reply:
We would lose support for IE 11 if we'd use `CustomEvent` instead of `initEvent`.

CustomEvent support:
![customevent](https://user-images.githubusercontent.com/20555112/73645081-e331d300-4676-11ea-8071-ddc1e9cedd94.png)

initEvent() support:
![initevent](https://user-images.githubusercontent.com/20555112/73645122-f8a6fd00-4676-11ea-801f-be53eeb98960.png)